test/unit/pages/prescription/PrescriptionForm.test.js

```diff
@@ -81,6 +81,10 @@ describe('PrescriptionForm', () => {
     expect(backButton).to.have.length(0);
   });
 
+  describe('clearCalculator', () => {
+
+  });
+
```

Comment:
Seems like this stub needs to be filled out?

Reply:
Exported separately and added test.